### DISCUSSION TITLE
Can you center the hover window vertically and horizontally center?

### DISCUSSION THREAD

**Discussion by nmsobri** [rmagatti/goto-preview#97] [Q&A] [answered]
As title, is it possible?

**Comment by rmagatti** [comment]
Hey, sorry, I'm not sure I understood the question.

**Reply by nmsobri** [reply]
Judging from the showcase, the preview window is always open to left of the cursor position.. can the preview window always open at the center of the screen regardless of the cursor position?

**Reply by rmagatti** [reply, marked as answer]
It can't currently. There's specific code in the plugin that specifically sets the windows based on the cursor position. I'd be happy to review a PR that adds support for ignoring the cursor position if that's something you're interested in.
